### DISCUSSION TITLE
tailwindcss-pseudo-elements support

### DISCUSSION THREAD

**Discussion by IRediTOTO** [ben-rogerson/twin.macro#237] [Ideas]
before / after very usefull to create complex design. Can you please add support for this pack ?

**Comment by ben-rogerson** [comment]
Twin has support for both the before and after pseudo elements. Just add them as variants and use the content class to add an empty content.

**Reply by IRediTOTO** [reply]
oh, i got it. Just didnt see hint with twin extention. :) THANK YOU
